linux/mcompile/keymap/keymap.cpp

```diff
@@ -352,45 +344,54 @@ int write_US_ToVector(vec_dword_3D& vec, std::string language, const char* secti
     return 1;
   }
 
-  fclose(fp);
+  if (vector_completeUS.size() != 48) {
+    printf("WARNING: the wrong keyboard input might have been chosen.\n");
+    return 0;
+  }
+
+  if (vector_completeUS.size() < 2) {
+    printf("ERROR: several keys of the US keyboard are not processed \n");
+    return 1;
+  }
```

Comment:
We will never get the error because it'll have output the warning and returned, so we don't need this block, or we'll have to check for `< 2` first.

Reply:
OK-moved block